streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -477,6 +476,9 @@ public K peekNextKey() {
             super(storeName, iter, serdes);
             iter.seek(serdes.rawKey(from));
             this.rawToKey = serdes.rawKey(to);
+            if (this.rawToKey == null) {
```

Comment:
Nope since `iter.seek` will throw the NPE. We could check too, but not necessary.